api/DTOs/ContactDTO.cs

```diff
@@ -1,17 +1,41 @@
-using System.ComponentModel.DataAnnotations;
+using Microsoft.AspNetCore.Mvc;
 using System.Text.Json.Serialization;
 
-public class ContactDTO : IDTO
+namespace DTO.Contact
 {
-    [Required]
-    [JsonPropertyName("name")]
-    public required string Name { get; set; }
+    [ApiExplorerSettings(IgnoreApi = true)]
+    public class ContactRequest : BaseDTO
+    {
+        [JsonPropertyName("name")]
+        public string? Name { get; set; }
+
+        [JsonPropertyName("phone")]
+        public string? Phone { get; set; }
+
+        [JsonPropertyName("email")]
+        public string? Email { get; set; }
+    }
+
     
-    [Required]
-    [JsonPropertyName("phone")]
-    public required string Phone { get; set; }
-
-    [Required]
-    [JsonPropertyName("email")]
-    public required string Email { get; set; }
-}
\ No newline at end of file
+    [ApiExplorerSettings(IgnoreApi = true)]
+    public class ContactResponse : BaseDTO
+    {
+        [JsonPropertyName("id")]
+        public Guid Id { get; set; }
+
+        [JsonPropertyName("name")]
+        public string? Name { get; set; }
+
+        [JsonPropertyName("phone")]
+        public string? Phone { get; set; }
+
+        [JsonPropertyName("email")]
+        public string? Email { get; set; }
+
+        [JsonPropertyName("created_at")]
+        public DateTime? CreatedAt { get; set; }
+
+        [JsonPropertyName("updated_at")]
+        public DateTime? UpdatedAt { get; set; }
+    }
```

Comment:
Overal waar je nu `ContactResponse` gaat gebruiken moeten de id created_at en updated_at ingevuld worden. Ik weet op dit moment niet zeker als dit gewenst is om te tonen in de response dus ik zou voor nu gewoon simpel alleen Name, Phone en Email als de `ContactResponse` zetten

![image](https://github.com/user-attachments/assets/2bf091bf-d023-472b-9392-b4046b37173b)